src/types/network-monitor.ts

```diff
@@ -109,9 +110,13 @@ export interface InterestingTransaction {
   receipt?: TransactionReceipt
   allLogs: Log[]
 }
+export interface ExtraDataType {
```

Comment:
Why is this ExtraData?

Reply:
This additional field serves as a repository for the CrossChainMessageType, and it's handy for adding any other extra data down the road.